### ISSUE TITLE
SelectorData Error in 3.4

### ISSUE THREAD

**Issue by jrock2004** [ember-a11y/ember-a11y-testing#128]
Getting a bunch of `Expect axe._selectorData to be set up` in the console. How can I tell why I am getting these? I get this when running ember -s and browing the first page I landed on. 

```
ENV['ember-a11y-testing'] = {
      componentOptions: {
        turnAuditOff: false, // Change to true to disable the audit in development
        axeOptions: {
          rules: {
            'region': { enabled: true } // enable region rule
          },
          checks: {
            'color-contrast': { options: { noScroll: true } } // disable scrolling of color-contrast check
          }
        },
        visualNoiseLevel: 2,
        axeViolationClassNames: ['alert-box', 'alert-box--a11y']
      }
    }
```

Did I miss something else I need to configure beyond installation to get this to work

**Comment by bahdvp** [comment]
I just experienced the same issue. Did you ever resolve this?

**Comment by jrock2004** [comment]
No, I stopped playing around for now

**Comment by drewlee** [comment]
@jrock2004 @bahdvp Are you still encountering this issue? If so, please provide the version of ember-a11y-testing and axe-core that your application resolves to, and steps or repo where the issue can be reliably reproduced. I'm not able to reproduce when setting up a new Ember v3.4 project.

**Comment by drewlee** [comment]
Based on reports, this issues seems to be localized to a very small set of users and we haven't received any means of reproducing it consistently. Will likely close the issue in a day or so due to lack of responses.
